### PR TITLE
Add man page for atomicapp

### PR DESCRIPTION
**Pull request by ingvagabund**
Content collected from atomicapp [command] -h and from docs.

### CODE REVIEW

**Comment by dustymabe** [comment]
hey @ingvagabund, thanks for the PR. What is the best way to generate this file when the help changes?

**Comment by ingvagabund** [comment]
As argparse does not provide man page generator (or could not find any post stating otherwise), I would write a simple script that will parse CLI.create_parser method. The code inside is written and structured pretty well.

Or create a patch that gets applied on CLI.create_parser method each time before the man page is generated and prints a list of all options in JSON. Which can be then easily transformed into man page.

**Comment by cdrage** [comment]
This is good for merging provided that #549 get's merged too :+1:

File: docs/man/atomicapp.1 [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+.TH "ATOMICAPP" "1" " atomicapp User Manuals" "" "Jan 2016"  ""
+
+.SH NAME
+.PP
+atomicapp \- Install and run an Atomic App, a containerized application conforming to the Nulecule Specification
+
+.SH SYNOPSIS
+.PP
+\fBatomicapp\fP [OPTIONS] {stop,run,genanswers,install}
```
Comment (cdrage) [inline docs/man/atomicapp.1]: install not included anymore, please replace with fetch

File: docs/man/atomicapp.1 [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+.SH DESCRIPTION
+.PP
+Atomic App is a reference implementation of the Nulecule Specification.
+It can be used to bootstrap container applications and to install
```
Comment (cdrage) [inline docs/man/atomicapp.1]: s/install/fetch/g

File: docs/man/atomicapp.1 [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+The provider to use. Overrides provider value in
+answerfile.
+
+.TP
+\fB\-\-ask\fP
+Ask for params even if the defaul value is provided
+
+.TP
+\fB\-\-destination\fP DESTINATION
+Destination directory for install. This defaults to a
```
Comment (cdrage) [inline docs/man/atomicapp.1]: fetch

File: docs/man/atomicapp.1 [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+.SS
+\fBStop options\fP
+
+.TP
+\fB\-\-provider\fP {docker,kubernetes,openshift,marathon}
+The provider to use. Overrides provider value in
+answerfile.
+
+.SS
+\fBInstall options\fP
```
Comment (cdrage) [inline docs/man/atomicapp.1]: fetch

File: docs/man/atomicapp.1 [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+\fB\-\-no\-deps\fP
+Skip pulling dependencies of the app
+
+.TP
+\fB\-u\fP, \fB\-\-update\fP
+Re-pull images and overwrite existing files
+
+.TP
+\fB\-\-destination\fP DESTINATION
+Destination directory for install. This defaults to a
```
Comment (cdrage) [inline docs/man/atomicapp.1]: fetch

**Comment by dustymabe** [comment]
hey @cdrage. let's figure out the best way to generate this information ourselves and then open a new PR and make sure that PR doesn't have any of these issues.

thanks again @ingvagabund

**Comment by cdrage** [comment]
Hi @ingvagabund thanks so much for the PR, we're going to find a way to generate this automatically similar to how https://github.com/projectatomic/atomic does it :)

**Comment by ingvagabund** [comment]
@cdrage, hope you will find fully automatic way guys!!! Np.

**Pull request closed without merging**